packages/roosterjs-content-model/lib/modelApi/common/retrieveModelFormatState.ts

```diff
@@ -59,6 +62,11 @@ export function retrieveModelFormatState(
                         segments.some(segment => segment.segmentType == 'Image');
 
                     isFirst = false;
+
+                    if (segment.segmentType === 'Image' && isFirstImage) {
```

Comment:
A question to PM: for segments format, if the selection contains multiple different format, such as different font names, we will return undefined for that format so that user know that there are different fonts selected. Do we want to do the same for image?